Hire-Wire/hire-wire
# Implemented the authetication feature

**What does this PR do?**

Build out the authentication feature for the hire-wire

**List of task completed**

-   [x] Resolved the duplicate, models, routes, controller files
-  [x] Built the User model 
-  [x] Implemented the routes for the user(register, login, logout, update, view, delete)
-  [x] Built the controllers to handle each of the logic for the user routes
-  [x] Installed the passport plugin to help with authentication and validation


**How can this be tested?**
How can this be tested?

-  Pull these changes:
-  Ensure your database connection is up to date
- Install dependencies using `npm install`
-  Run `npx sequelize-cli db:migrate` to create the User table in your database
-  Run `npm start ` to start the application
-  Go to your web browser and enter http://localhost:8000/

- Highlight how this PR can be tested

Run `npm test`

**Additional info**

Add more info if any. eg fixes #.
